File: SDL2.OC/SDL2.OC/ViewController.m
//
//  ViewController.m
//  SDL2.OC
//
//  Created by Dream on 2021/6/26.
//

#import "ViewController.h"

#import <SDL2/SDL.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    SDL_Init(SDL_INIT_AUDIO);
    SDL_version version;
    NSLog(@"%hhu.%hhu.%hhu", version.minor, version.major, version.patch);
    SDL_GetVersion(&version);
    NSLog(@"%hhu.%hhu.%hhu", version.minor, version.major, version.patch);
}


@end

File: Demo/SDL2.Swift/SDL2.Swift/ViewController.swift
//
//  ViewController.swift
//  SDL2.Swift
//
//  Created by Dream on 2021/7/10.
//

import UIKit

import SDL2

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        SDL_Init(SDL_INIT_AUDIO)
        var version = SDL_version()
        print("\(version.major).\(version.minor).\(version.patch)")
        SDL_GetVersion(&version)
        print("\(version.major).\(version.minor).\(version.patch)")
    }
}

